src/components/seo.tsx

```diff
@@ -1,70 +1,68 @@
-/** @jsxImportSource @emotion/react */
```

Comment:
What's going on with the diff in this file? Did it maybe change EOL characters or something? It doesn't look like anything changed aside of removal of the pragma but the entire file is in the diff.